src/inc/regdisp.h

```diff
@@ -8,7 +8,7 @@
 
 #ifdef DEBUG_REGDISPLAY
 class Thread;
-
+struct REGDISPLAY;
```

Comment:
My guess is that for other targets, the REGDISPLAY is actually always forward declared before including this file and ARM64 is different. Since ARM32, x64 and x86 have all passed the CI.